### ISSUE TITLE
kcli delete plan doesn't delete dns entries defined in a plan

### ISSUE THREAD

**Issue by palonsoro** [karmab/kcli#695]
If I define dns entries in a plan, those get created as expected but when the plan is deleted with `kcli delete plan`, those are never deleted, leaving stale DNS entries.

**Comment by palonsoro** [comment]
Possible way to go: Record in network metadata which hostname-ip DNS associations are managed by kcli under each plan and look for those while deleting. Not perfect, but at least better than nothing.

**Comment by karmab** [comment]
while it can certainly be improved, not that when recreating those dns entries, they get updated with the correct ips if necessary

**Comment by palonsoro** [comment]
Even if created under a different plan? Regardless of whether what changes is the ip or the host part?

**Comment by karmab** [comment]
yes!

**Comment by palonsoro** [comment]
Ok. I have had problems in the past with the leaked DNS entry, but I cannot tell if it was with a kcli command or a script of my own.

But anyway, even if not high prio, it should be good to avoid the leaks anyway.

**Comment by karmab** [comment]
TLDR: wontdo

I had a go at this but I cant store the plan as metadata for dns entries

The dns entry is stored like this for kvm

```
    <host ip='192.168.130.23'>
      <hostname>lab-mgmt-spoke1-node-0.karmalabs.corp</hostname>
      <hostname>api.mgmt-spoke1.karmalabs.corp</hostname>
      <hostname>api-int.mgmt-spoke1.karmalabs.corp</hostname>
    </host>
```

I tried to put the plan as extra attribute of the host tag, but it gets (silently) dropped by libvirt.
So I'll keep current behaviour for now
